metrics/web/lib/common/presentation/tooltip_icon/widgets/tooltip_icon.dart

```diff
@@ -0,0 +1,64 @@
+import 'package:flutter/material.dart';
+import 'package:flutter/rendering.dart';
+import 'package:metrics/base/presentation/widgets/base_popup.dart';
+import 'package:metrics/base/presentation/widgets/svg_image.dart';
+import 'package:metrics/common/presentation/metrics_theme/widgets/metrics_theme.dart';
+import 'package:metrics/common/presentation/tooltip_popup/widgets/tooltip_popup.dart';
+
+/// A widget with a tooltip icon that displays the [TooltipPopup] on hover.
+class TooltipIcon extends StatelessWidget {
+  /// A width of the [TooltipPopup].
+  static const double _tooltipPopupWidth = 187.0;
+
+  /// A tooltip text to display in the [TooltipPopup].
+  final String tooltip;
+
+  /// A source of the image to display.
+  final String src;
+
+  /// Creates a new instance of the [TooltipIcon].
+  ///
+  /// The [tooltip] must not be `null`.
+  const TooltipIcon(
+    this.src, {
```

Comment:
Should we assert that this parameter should not be null? Also, maybe it will be better to make it a named parameter?